src/main/java/io/xpring/payid/PayIDClient.java

```diff
@@ -0,0 +1,120 @@
+package io.xpring.payid;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.gson.reflect.TypeToken;
+import io.xpring.payid.generated.*;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.interledger.spsp.PaymentPointer;
+
+import io.xpring.payid.generated.api.DefaultApi;
+import io.xpring.payid.generated.model.PaymentInformation;
+
+import java.io.IOException;
+import java.lang.reflect.Type;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Implements interaction with a PayID service.
+ * Warning:  This class is experimental and should not be used in production applications.
+ */
+public class PayIDClient {
+    /**
+     * The network this PayID client resolves on.
+     */
+    private XRPLNetwork network;
+
+    /**
+     * Whether to enable SSL Verification.
+     */
+    private boolean enableSSLVerification;
+
+    /**
+     * Initialize a new PayIDClient.
+     *
+     * @param network The network that addresses will be resolved on.
+     */
+    public PayIDClient(XRPLNetwork network) {
+        this.network = network;
+        this.enableSSLVerification = true;
+    }
+
+    /**
+     * Set whether to enable or disable SSL verification.
+     *
+     * Exposed for testing purposes.
+     */
+    public void setEnableSSLVerification(boolean enableSSLVerification) {
+        this.enableSSLVerification = enableSSLVerification;
+    }
+
+    /**
+     * Resolve the given PayID to an XRP Address.
+     *
+     * Note: The returned value will always be in an X-Address format.
+     *
+     * @param payID The payID to resolve for an address.
+     * @return An XRP address representing the given PayID.
+     */
+    public String xrpAddressForPayID(String payID) throws PayIDException {
+        PaymentPointer paymentPointer = PayIDUtils.parsePayID(payID);
+        if (paymentPointer == null) {
+            throw PayIDException.invalidPaymentPointerExpection;
+        }
+
+        ApiClient apiClient = new ApiClient();
+        apiClient.setBasePath("https://" + paymentPointer.host());
+        apiClient.setVerifyingSsl(enableSSLVerification);
+
+        String path = paymentPointer.path().substring(1);
+        final String[] localVarAccepts = {
+                "application/xrpl-" + this.network.getNetworkName() + "+json"
+        };
+
+        // NOTE: Swagger produces a higher level client that does not require this level of configuration,
+        // however access to Accept headers is not available unless we access the underlying class.
+        // TODO(keefertaylor): Factor this out to a helper function to make it clear which inputs matter.
+
+        String localVarPath = "/{path}"
+                .replaceAll("\\{" + "path" + "\\}", apiClient.escapeString(path.toString()));
```

Comment:
Isn't this equivalent to:
`String localVarPath = "/" +  apiClient.escapeString(path.toString());`

Reply:
Yeah. I copy / pasted this out of a generated file. Obviously the generated code is more verbose / formulaic. Added your method.